awx/main/tasks.py

```diff
@@ -1303,9 +1364,9 @@ def build_args(self, job, **kwargs):
 
         ssh_username, become_username, become_method = '', '', ''
         if creds:
-            ssh_username = kwargs.get('username', creds.get_input('username', default=''))
-            become_method = kwargs.get('become_method', creds.get_input('become_method', default=''))
-            become_username = kwargs.get('become_username', creds.get_input('become_username', default=''))
+            ssh_username = creds.username
```

Comment:
Let's continue using `.get_input()`, this magic attribute access is going away in the near future.

Reply:
yeah this is pretty important to resolve in the diff before merge

Reply:
Would like to have this addressed pre-merge.